ocp_resources/sriov_network_node_state.py

```diff
@@ -9,6 +9,9 @@ class SriovNetworkNodeState(NamespacedResource):
 
     api_group = NamespacedResource.ApiGroup.SRIOVNETWORK_OPENSHIFT_IO
 
+    class SyncStatus:
+        INPROGRESS = "InProgress"
```

Comment:
Please add typing.
`INPROGRESS: str = "InProgress"`

Reply:
and please split the attr name `IN_PROGRESS`

Reply:
Done.